### DISCUSSION TITLE
Windows cannot access the specified device, path, or file error when trying to open installer

### DISCUSSION THREAD

**Discussion by xtreme987** [rocksdanister/lively#579] [Q&A]
Trying to install version 1.700 but I get this message - 

![lively error](https://user-images.githubusercontent.com/86535828/123525393-4d00b180-d69e-11eb-9b0a-9946aa28493c.jpg)

**Comment by rocksdanister** [comment]
I found this when searching:
https://support.microsoft.com/en-us/topic/-windows-cannot-access-the-specified-device-path-or-file-error-when-you-try-to-install-update-or-start-a-program-or-file-46361133-47ed-6967-c13e-e75d3cc29657

**Reply by xtreme987** [reply]
Okay after completely shutting my computer & rebooting I was able to finally install however AVG detected that a virus "IDP.Generic" was found. It said it quarantined it. Now Lively isn't working again. Any further help ?

**Reply by rocksdanister** [reply]
Duplicate: https://github.com/rocksdanister/lively/issues/358
